### PR TITLE
Feature/password hashing

### PR DESCRIPTION
**Pull request by mattong**
image

### CODE REVIEW

File: lib/birdie/accounts/user.ex [hunk 1]
```diff
@@ -22,4 +28,9 @@ defmodule Birdie.Accounts.User do
     |> cast(attrs, [:username, :password])
```
Comment (raaims) [inline lib/birdie/accounts/user.ex]: 😠

File: lib/birdie_web/router.ex [hunk 2]
```diff
@@ -17,13 +17,13 @@ defmodule BirdieWeb.Router do
   scope "/", BirdieWeb do
     pipe_through :browser # Use the default browser stack
 
-    get "/", PageController, :index
+    # get "/", PageController, :index
```
Comment (raaims) [inline lib/birdie_web/router.ex:20]: delete if you don't need it

File: lib/birdie/accounts/user.ex [hunk 3]
```diff
@@ -5,12 +5,18 @@ defmodule Birdie.Accounts.User do
 
   schema "users" do
     field :password, :string, virtual: true
```
Comment (raaims) [inline lib/birdie/accounts/user.ex:7]: we store unencrpyted password?

**Review by raaims — CHANGES_REQUESTED** [review]
😠

File: lib/birdie/accounts/user.ex (hunk 3 above)
Comment (mattong) [reply lib/birdie/accounts/user.ex:7]: we don;t. Pumapasok siya as nil.

File: lib/birdie/accounts/user.ex (hunk 1 above)
Comment (mattong) [reply lib/birdie/accounts/user.ex]: Actually may paggagamitan pa ako nito huehue

**Review by raaims — APPROVED** [review]

**Pull request merged**